src/CCFieldValidator.js

```diff
@@ -2,11 +2,12 @@ import valid from "card-validator";
 import pick from "lodash.pick";
 import values from "lodash.values";
 import every from "lodash.every";
-import { CARDS_OVERRIDES, VALID, INCOMPLETE } from "./constants";
+import { CARDS_OVERRIDES, VALID, INCOMPLETE, CABAL_CARD_TYPE } from "./constants";
 import { toStatus, maestroCardStatus } from "./utils";
 
 
 CARDS_OVERRIDES.forEach(card => valid.creditCardType.addCard(card));
+valid.creditCardType.addCard(CABAL_CARD_TYPE);
```

Comment:
I really don't get it, why you treat CABAL_CARD_TYPE different?

Why don't you just add it to CARD_OVERRIDES array?

Reply:
Because it's not an override, it's a new card. Semantically speaking, it's not the same, even though they both use the addCard function.
The overrides do exactly that, they are meant to replace already existing credit cards. The cabal credit card doesn't exist in the library, that's why I did it separately.

Reply:
Well, change the name  if you don't like it. The action you are doing is the same, so I think you should do it only once.

Call it "ADDITIONAL_CARDS", "CARD_TO_BE_ADDED", "CARDS_OVERRIDES_OR_NEW_CARDS", or whatever you want.

Reply:
Ok, ok, I will!.